pkg/hash/hash_test.go

```diff
@@ -0,0 +1,80 @@
+package hash_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift-psap/special-resource-operator/pkg/hash"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+const emptyHash = "12161962213042174405"
+
+func TestHash(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Hash Suite")
+}
+
+var _ = Describe("Hash", func() {
```

Comment:
it is possible, but usually describe is not used under describe
In this case you can define Describe per File, and then add a Context per each function you are test, and under each Context you can add It per each specific scenario. Seems a little bit more logical to me

Reply:
Okay - I'll switch the per-function `Describe`s to `Context`s.